recipe/meta.yaml

```diff
@@ -20,7 +20,10 @@ requirements:
     - cython                                 # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
     - pybind11                               # [build_platform != target_platform]
-    - {{ compiler('fortran') }}
+    - {{ compiler('fortran') }}   # [unix]
+    # WARNING: It's not recommended to use these MinGW compilers with python extensions
+    # numpy.distutils has a complex mechanism to facilitate mixing gfortran and MSVC
```

Comment:
Maybe useful as a reference: https://pav.iki.fi/blog/2017-10-08/pywingfortran.html#building-python-wheels-with-fortran-for-windows